### PR TITLE
Make all "Action" serializers consistent

### PR DESCRIPTION
**Pull request by nickcatal**
We were repeating ourselves quite a bit inside serializers that all created new Actions. This streamlines things a bit and makes it easier to flesh out Action functionality down the line without rewriting a bunch of serializers.

### CODE REVIEW

File: app/action/serializers.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def __init__(self, instance=None, data=empty, **kwargs) -> None:
+        self.incomplete = kwargs.pop("incomplete", False)
+        super().__init__(instance=instance, data=data, **kwargs)
+
+    def create(self, validated_data: Dict[(str, Any)]) -> "Model":
+        validated_data["action"] = Action.objects.create()
+        return super().create(validated_data)
+
+    def get_field_names(
+        self, declared_fields: Mapping[(str, serializers.Field)], info: "FieldInfo"
```
Comment (jlev) [inline app/action/serializers.py:31]: Do linting rules suggest putting these on newlines? I know types is the new python way, but I find this hard to parse.

Comment (nickcatal) [reply app/action/serializers.py:31]: It's what the linter says
